samples/apps/autogen-studio/frontend/src/components/atoms.tsx

```diff
@@ -722,7 +766,7 @@ export const ModelSelector = ({
 
   const handleOk = () => {
     if (newModelConfig?.model.trim()) {
-      const sanitizedConfig = sanitizeModelConfig(newModelConfig);
+      const sanitizedConfig = newModelConfig;
```

Comment:
just want to confirm `newModelConfig` has been sanitized?

Reply:
Oh yes .. model sanitization logic has been moved to the backend, in workflowmanager.
Here, sanitization just means we remove any extra fields from the model object (e.g. things like id, timestamp, description etc).
This enforces sanitizated behavior at the backend. 
That is why sanitization is not needed on the frontend anymore.